### PR TITLE
Issue #34 remove gmail service wip

### LINKED ISSUE #34: Remove GMail based mail service
Remove GMail mail service and all related data as it's not needed anymore

Sabtasks:
- [x] #116 Find all connection to GMail service
- [x] #118 Remove GMail services and all related date

### PR DESCRIPTION
**Pull request by Myroslavv**
### Summary
Discard of Gmail service in VolleyManagement.Services.
Delete part of RegisterDependencies in src/VolleyManagement.Backend/VolleyManagement.UI/Infrastructure/IOCUIModule.cs which is responsible for bundling GmailAccountMailService to IMailService.

closes #34 

### Areas Of Interest
This might be too easy. It works well after just deleting one file and minory editing another.
Should we get rid of something else?

### Checklist
- Find file which's responsible for Gmail service.
- Delete file.
- Solve occured errors and warnings.

#### Design
Nothing.

#### Perfromance
Nothing.

#### Unit Tests
Nothing

### CODE REVIEW

File: src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Admin/Views/Dashboard/Index.cshtml [hunk 1]
```diff
@@ -3,5 +3,4 @@
     ViewBag.Title = "Admin Dashboard";
 }
 
-<h2>Admin Dashboard</h2>
```
Comment (sdiachen) [inline src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Admin/Views/Dashboard/Index.cshtml]: How does this change relate to the task at hand?

File: src/VolleyManagement.Backend/VolleyManagement.UI/Infrastructure/IOCUIModule.cs [hunk 2]
```diff
@@ -24,10 +24,6 @@ public void RegisterDependencies(IIocContainer container)
             {
                 container.Register<IMailService, DebugMailService>(IocLifetimeEnum.Scoped);
             }
-            else if (Is<IISDeployment>.Enabled)
-            {
-                container.Register<IMailService, GmailAccountMailService>(IocLifetimeEnum.Scoped);
-            }
             else
             {
                 container.Register<IMailService, SendGridMailService>(IocLifetimeEnum.Scoped);
```
Comment (sdiachen) [inline src/VolleyManagement.Backend/VolleyManagement.UI/Infrastructure/IOCUIModule.cs]: This was not in AC but SendGrid should be active only when `Is<IISDeployment>.Disabled`

File: src/VolleyManagement.Backend/VolleyManagement.UI/Views/Home/Index.cshtml [hunk 3]
```diff
@@ -5,4 +5,4 @@
 <h2>Index</h2>
 <li>@Html.ActionLink("MVC", "Index", "Tournaments", new { area = "Mvc" }, new { })</li>
 
-<li>@Html.ActionLink("Web API", "Index", new { area = "WebApi" })</li>
```
Comment (sdiachen) [inline src/VolleyManagement.Backend/VolleyManagement.UI/Views/Home/Index.cshtml]: How does this change relate to the task?

File: src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Admin/Views/Dashboard/Index.cshtml (hunk 1 above)
Comment (ViznyakS) [reply src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Admin/Views/Dashboard/Index.cshtml]: No, this cahnge was made before solving the issue

Comment (sdiachen) [reply src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Admin/Views/Dashboard/Index.cshtml]: So it should not be part of this PR

File: src/VolleyManagement.Backend/VolleyManagement.UI/Views/Home/Index.cshtml (hunk 3 above)
Comment (Myroslavv) [reply src/VolleyManagement.Backend/VolleyManagement.UI/Views/Home/Index.cshtml]: This doesn't.

File: src/VolleyManagement.Backend/VolleyManagement.UI/Infrastructure/IOCUIModule.cs (hunk 2 above)
Comment (MikolaB) [reply src/VolleyManagement.Backend/VolleyManagement.UI/Infrastructure/IOCUIModule.cs]: http://www.innolution.com/resources/glossary/acceptance-criteria

**Pull request closed without merging**